tests/unit/Hooks/ImageHooksTest.php

```diff
@@ -99,19 +102,18 @@ public function testPostCreate() {
 		$this->logger->log('debug', 'Test Image Height: '.$imageHeight, array('app' => 'MediaMetadata'));
 		$this->logger->log('debug', 'Test Image Width: '.$imageWidth, array('app' => 'MediaMetadata'));
 
-		$this->imageDimension->expects($this->once())
-			->method('setImageId')
-			->with($jpgFile->getId());
-		$this->imageDimension->expects($this->once())
-			->method('setImageHeight')
-			->with($imageHeight);
-		$this->imageDimension->expects($this->once())
-			->method('setImageWidth')
-			->with($imageWidth);
-
-		$this->mapper->expects($this->once())
-			->method('insert')
-			->with($this->imageDimension);
+		$this->db->expects($this->once())
+			->method('lastInsertId')
+			->with($this->equalTo('*PREFIX*table'));
+
+		$this->imageDimension->setId($jpgFile->getId());
+		$this->assertContains('id', $this->imageDimension->getUpdatedFields());
+		$this->imageDimension->setImageHeight($imageHeight);
+		$this->assertEquals(2, count($this->imageDimension->getUpdatedFields()));
```

Comment:
This is too vague, you need to check the values of the row. In the example you gave me, it shows that a new fake TestEntity class was created in order to test the insertion of rows. You should try to do the same.